packages/analytics/ios/RNFBAnalytics/RNFBAnalyticsModule.m

```diff
@@ -142,6 +142,22 @@ - (dispatch_queue_t)methodQueue {
       return resolve([FIRAnalytics appInstanceID]);
   }
 
+  RCT_EXPORT_METHOD(setDefaultEventParameters:
+    params:
+        (NSDictionary *) params
+        resolver:
+        (RCTPromiseResolveBlock) resolve
+        rejecter:
+        (RCTPromiseRejectBlock) reject) {
+    @try {
+      [FIRAnalytics setDefaultEventParameters:parameters:[self cleanJavascriptParams:params]];
```

Comment:
I think the API call needs to look like this - and I *think* I can commit the suggestion so CI will chew on it and validate my assertion

```suggestion
      [FIRAnalytics setDefaultEventParameters:[self cleanJavascriptParams:params]];
```